www/include/configuration/configObject/traps/formTraps.php

```diff
@@ -166,15 +180,32 @@ function myReplace()
  * Generic fields
  */
 $form->addElement('text', 'traps_oid', _("OID"), $attrsText);
-$form->addElement('select', 'traps_status', _("Default Status"), array(0=>_("Ok"), 1=>_("Warning"), 2=>_("Critical"), 3=>_("Unknown")), array('id' => 'trapStatus'));
+$form->addElement(
+    'select',
+    'traps_status',
+    _("Default Status"),
+    array(
+        0=>_("Ok"),
+        1=>_("Warning"),
+        2=>_("Critical"),
+        3=>_("Unknown")
```

Comment:
Add whitespace before & after the key, value 

```suggestion
         3 => _("Unknown")
```